plugins/main/ptp/ptp.go

```diff
@@ -84,6 +84,11 @@ func setupContainerVeth(netns ns.NetNS, ifName string, mtu int, pr *current.Resu
 			return err
 		}
 
+		// Wait for addresses to leave tentative state (for up to 10 seconds)
+		if err := ip.SettleAddresses(ifName, 10); err != nil {
```

Comment:
Should we just make ConfigureIface() handle the settling?  That might reduce code in the plugins a bit; we'd still need the Settle call for standalone IP assignment like bridge.go does though.